src/browser/jotz_browser.js

```diff
@@ -100,6 +105,12 @@ var JotzBrowser = Backbone.Model.extend({
       e.sender.send('destroy-note-reply', err);
     });
   },
+  destroyNotesByNbId: function(e, noteIds) {
```

Comment:
can the iteration over the noteIds be encapsulated in the NotesAPI? I suggest: `NotesAPI.destroyNotes`.

Please either remove or use the commented out reply to the renderer on line 112.